integration/src/main/java/scripts/testscripts/PrivateControlledGcsBucketLifecycle.java

```diff
@@ -152,26 +151,23 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     assertEquals(createdBlob, retrievedBlob);
     logger.info("Private resource user can read {} from bucket", retrievedBlob.getName());
 
-    // TODO(PF-633): workspace owners can read from private buckets via "roles/storage.admin" and
-    //  "roles/viewer"
-    // // Owner cannot read the bucket contents
-    // StorageException ownerCannotReadPrivateBucket =
-    //     assertThrows(
-    //         StorageException.class,
-    //         () -> ownerStorageClient.get(blobId),
-    //         "Workspace owner was able to read private bucket");
-    // assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, ownerCannotReadPrivateBucket.getCode());
-    // logger.info("Workspace owner cannot read from private bucket as expected");
-
-    // TODO(PF-633): workspace readers can read from private buckets via "roles/viewer".
-    // // Workspace reader also cannot read private bucket contents
-    // StorageException readerCannotReadPrivateBucket =
-    //     assertThrows(
-    //         StorageException.class,
-    //         () -> workspaceReaderStorageClient.get(blobId),
-    //         "Workspace reader was able to read private bucket");
-    // assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, readerCannotReadPrivateBucket.getCode());
-    // logger.info("Workspace reader cannot read from private bucket as expected");
+    // Owner cannot read the bucket contents
+    StorageException ownerCannotReadPrivateBucket =
+        assertThrows(
+            StorageException.class,
+            () -> ownerStorageClient.get(blobId),
+            "Workspace owner was able to read private bucket");
+    assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, ownerCannotReadPrivateBucket.getCode());
+    logger.info("Workspace owner cannot read from private bucket as expected");
+
+    // Workspace reader also cannot read private bucket contents
```

Comment:
Just curious, in this class I see you're testing owners and readers -- how come not writers also?

Reply:
No strong reason. Workspace owners already have a superset of writer permission so I think it would be fairly low signal and the test is reasonably long already, but that's pretty much it.